pandas/tests/series/test_analytics.py

```diff
@@ -1788,3 +1788,11 @@ def test_value_counts_categorical_not_ordered(self):
                         index=exp_idx, name='xxx')
         tm.assert_series_equal(s.value_counts(normalize=True), exp)
         tm.assert_series_equal(idx.value_counts(normalize=True), exp)
+
+    def test_nat_operations(self):
```

Comment:
yep, I was not sure where to put this. Finding a place to put a test it a good way to get lost :-)

but note that it is *not* a timeseries (in the sense of a series having a datetime index), so it would also be strange to put it in `tseries/test_timeseries`.